### PR TITLE
Fixes: cannot prune builds on buildConfig change

### PR DESCRIPTION
**Pull request by coreydaley**
https://bugzilla.redhat.com/show_bug.cgi?id=1549832

### CODE REVIEW

**Comment by coreydaley** [comment]
@openshift/sig-developer-experience ptal

File: pkg/cmd/server/bootstrappolicy/controller_policy.go [hunk 1]
```diff
@@ -120,6 +120,7 @@ func init() {
 		Rules: []rbac.PolicyRule{
 			rbac.NewRule("get", "list", "watch").Groups(buildGroup, legacyBuildGroup).Resources("buildconfigs").RuleOrDie(),
 			rbac.NewRule("create").Groups(buildGroup, legacyBuildGroup).Resources("buildconfigs/instantiate").RuleOrDie(),
+			rbac.NewRule("delete").Groups(buildGroup, legacyBuildGroup).Resources("builds").RuleOrDie(),
```
Comment (bparees) [inline pkg/cmd/server/bootstrappolicy/controller_policy.go]: why doesn't the build-controller also need this permission?  doesn't it delete builds also?

Comment (bparees) [reply pkg/cmd/server/bootstrappolicy/controller_policy.go]: nm, i'm blindish.

**Comment by bparees** [comment]
/lgtm

will need to be cherry-picked against release-3.9 though, since we already branched.

**Comment by bparees** [comment]
(holding since master is closed)

**Pull request merged**